test/bdd/steps/network.js

```diff
@@ -13,8 +13,9 @@ const { sha3_256 } = require('js-sha3');
 const { deepEqual } = require('jsprim');
 
 const OtNode = require('./lib/otnode');
+const Utilities = require('../../../modules/Utilities');
 const LocalBlockchain = require('./lib/local-blockchain');
-const httpApiHelper = require('./lib/http-api-helper');
+const { apiImportInfo, apiFingerprint } = require('./lib/http-api-helper');
```

Comment:
I originally left `httpApiHelper` instead extracting function since I suppose it's going to grow a lot. Image how it's going to look when using object destructing.

Reply:
valid point, thanks for pointing that to me. done (i mean I reverted back to httpApiHelper )